.eslintrc.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
Seems like there are few rules from the old files missing here

Reply:
the import rule is handled implicitly now (i tested with adding a rando import). 
Jest is added to the spec file config

I haven't checked the node one though, but not sure it is needed (which node scripts are we worried about?) also the plugin hasn't been updated since 2020... maybe this is for e2e???